packages/server/graphql/public/typeDefs/RegenerateOAuthAPIProviderSecretInput.graphql

```diff
@@ -0,0 +1,3 @@
+input RegenerateOAuthAPIProviderSecretInput {
+  providerId: ID!
```

Comment:
+1 This isn't wrong, but it's a good example of why I'm not a huge fan of input objects when we don't need them. It means we create more types & have extra nesting. Mutations accept multiple args!